### PR TITLE
Updates Native Distributions documentation

### PR DESCRIPTION
**Pull request by cjbrooks12**
Updates Native Distributions documentation to include a note about Java Modules, and how to determine necessary modules and provide them to the Gradle plugin configuration

### CODE REVIEW

File: tutorials/Native_distributions_and_local_execution/README.md [hunk 1]
```diff
@@ -36,6 +39,10 @@ compose.desktop {
         mainClass = "example.MainKt"
 
         nativeDistributions {
+            modules(
+                "java.base",
```
Comment (olonho) [inline tutorials/Native_distributions_and_local_execution/README.md:43]: Those modules are actually included by default.

Comment (olonho) [reply tutorials/Native_distributions_and_local_execution/README.md:43]: Maybe use smth like `java.sql` instead.

File: tutorials/Native_distributions_and_local_execution/README.md [hunk 2]
```diff
@@ -57,13 +64,18 @@ containing the `main` function.
...
 To run a final binary image, use `runDistributable` instead.
-* `createDistributable` is used to create a prepackaged application image a final application image without creating an installer. 
+* `createDistributable` is used to create a prepackaged application image a final application image without creating an installer.
 * `runDistributable` is used to run a prepackaged application image. 
   
 Note, that the tasks are created only if the `application` block/property is used in a script.
 
 After a build, output binaries can be found in `${project.buildDir}/compose/binaries`.
 
+At this time, the Gradle plugin does not automatically determine the JDK Modules necessary to run, and you must manually
```
Comment (olonho) [inline tutorials/Native_distributions_and_local_execution/README.md:74]: This got to be updated per https://github.com/JetBrains/compose-jb/pull/501

File: tutorials/Native_distributions_and_local_execution/README.md [hunk 3]
```diff
@@ -8,7 +8,10 @@ We will also demonstrate how to run an application locally with the same setting
 ## Gradle plugin
 
 `org.jetbrains.compose` Gradle plugin simplifies the packaging of applications into native distributions and running an application locally.
-Currently, the plugin uses [jpackage](https://openjdk.java.net/jeps/343) for packaging self-contained applications.
+Currently, the plugin uses [jpackage](https://openjdk.java.net/jeps/343) for packaging self-contained applications. Distributable 
+applications are self-contained, installable binaries which include all the Java runtime components they need, without requiring an installed
```
Comment (AlexeyTsvetkov) [inline tutorials/Native_distributions_and_local_execution/README.md:12]: > packaging self-contained applications. Distributable applications are self-contained

Minor: I think mentioning, that applications "self-contained" twice is a bit excessive

Comment (AlexeyTsvetkov) [inline tutorials/Native_distributions_and_local_execution/README.md:12]: Minor formatting: the line is too long (ideally it should be < 110 chars

**Review by AlexeyTsvetkov — APPROVED** [review]

**Comment by AlexeyTsvetkov** [comment]
Merged manually with minor changes (which resolve the review comments above).

@cjbrooks12 thanks for the contribution!

**Pull request closed without merging**